agent/taskresource/fsxwindowsfileserver/fsxwindowsfileserver_windows.go

```diff
@@ -552,18 +552,25 @@ func (fv *FSxWindowsFileServerResource) performHostMount(remotePath string, user
 
 	// New-SmbGlobalMapping cmdlet creates an SMB mapping between the container instance
 	// and SMB share (FSx for Windows File Server file-system)
-	cmd := execCommand("powershell.exe",
+
+	args := []string{
 		"New-SmbGlobalMapping",
 		localPathArg,
 		remotePathArg,
 		creds,
 		"-Persistent $true",
 		"-RequirePrivacy $true",
-		"-ErrorAction Stop")
+		"-ErrorAction Stop",
+	}
+	log_args := make([]string, len(args))
+	log_args = append(log_args, args...)
+	log_args[3] = "<creds>" // Clear the creds for logging.
```

Comment:
I think the debug log below would include cred like below
```
-Credential $(New-Object System.Management.Automation.PSCredential(\"...", $(ConvertTo-SecureString \"...\" -AsPlainText -Force)))
```

Reply:
Hey I removed the extra copy. Did you mean the error log shows it still? The index is skipped in the debug log so it wouldnt have it.